tests/topotests/bgp-default-ipv4-ipv6-unicast/test_bgp-default-ipv4-ipv6-unicast.py

```diff
@@ -34,6 +34,8 @@
 import pytest
 import functools
 
+pytestmark = pytest.mark.bgpd
```

Comment:
can we modify this to `pytestmark = [ pytest.mark.bgpd ]`

to follow the syntax in other files?

Reply:
I'll push this in once it is done

Reply:
Fixed.